src/realm/sync/noinst/client_impl_base.cpp

```diff
@@ -1738,6 +1738,7 @@ void Session::initiate_deactivation()
     logger.debug("Initiating deactivation"); // Throws
 
     m_state = Deactivating;
+    call_debug_hook(SyncClientHookEvent::SessionDeactivated);
```

Comment:
not really deactivated at this point

Reply:
I'll update the event name to `SessionDeactivating`